lib/hsts.c

```diff
@@ -54,7 +54,7 @@
 #define MAX_HSTS_DATELENSTR "64"
 #define UNLIMITED "unlimited"
 
-#ifdef DEBUGBUILD
+#if defined(DEBUGBUILD) || defined(UNITTESTS)
```

Comment:
Why is this needed?

The unit tests assume that `DEBUGBUILD` is defined as a prereq for running. This change seems to say that you can build with `UNITTESTS` without `DEBUGBUILD` ?

Reply:
Unit tests (and any other test) are _building_ fine without either `DEBUGBUILD` or `CURLDEBUG`. (That is after fixing the places where `DEBUGBUILD` was used to enable functions for unit tests.)

This is an outlier case where the `deltatime` variable of a debug feature (`CURL_TIME`) is actually required for building a unit test.